packages/quantic/playwright/page-object/configurationObject.ts

```diff
@@ -6,19 +6,19 @@ export class ConfigurationObject {
   }
 
   get tryButton(): Locator {
-    return this.page.locator('lightning-button[data-cy="cfg-try"]');
+    return this.page.locator('lightning-button[data-testid="cfg-try"]');
   }
 
   get resetButton(): Locator {
-    return this.page.locator('lightning-button[data-cy="cfg-reset"]');
+    return this.page.locator('lightning-button[data-testid="cfg-reset"]');
   }
 
   get configurationForm(): Locator {
     return this.page.locator('slot[name="configuration"]');
   }
 
   private getInputSelector(field: string): string {
-    return `lightning-input[data-cy="cfg-${field}"] input`;
+    return `lightning-input[data-testid="cfg-${field}"] input`;
```

Comment:
Playwright seems to recommend to replace `page.fill` with a `locator.fill`

So we could change instead the `this.getInputSelector(key)` to a `this.getInputByKey(key).fill()`

```
private getInputByField(field:string): Locator {
    return this.page.getByTestId(`cfg-${field}`).locator('input');
}
```

Something like that instead.